specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -103,6 +103,17 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "$ref": "#/definitions/Debug",
+            "name": "debug",
+            "in": "query",
+            "type": "string",
```

Comment:
Should this be an enum?

Reply:
The $ref (which is an enum) should take care of that, right?

Reply:
Should it be similar to queryType where the enum is inlined?

Reply:
I'm not sure -- I tried to use $ref here for reusability, but does it result in a material difference?

Reply:
I dug a little into the swagger docs, and it seems the $ref should be fine and is the preferred route for reusability. As for enum concerns, I'll go ahead and remove the enum portion from "Debug". It's worth pointing out that we've documented Answers / Captions as enums, and those features are already live - maybe that was incorrect.